src/handlers/thirdparty.ts

```diff
@@ -34,4 +36,37 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
 
   let metric = buildCountFromThirdParty()
   metric.save()
-}
\ No newline at end of file
+}
+
+export function handleItemUpdated(event: ItemUpdated): void {
+  let item = Item.load(event.params._itemId)
+  if (item == null) {
+    return
+  }
+
+  item.metadata = event.params._metadata
```

Comment:
The event params should be the `rawMetadata` no the metadata object itself. The metadata object must be built on each update